src/components/application_manager/test/commands/mobile/alert_request_test.cc

```diff
@@ -63,10 +66,55 @@ using am::commands::AlertRequest;
 using am::commands::MessageSharedPtr;
 using am::event_engine::Event;
 using am::MockMessageHelper;
+using policy_test::MockPolicyHandlerInterface;
 
 typedef SharedPtr<AlertRequest> CommandPtr;
+namespace {
+const uint32_t kDefaultTimeout = 1000u;
+const uint32_t kConnectionKey = 1u;
+const uint32_t kCorrelationId = 2u;
+const uint32_t kFunctionId = 3u;
+}  // namespace
 
 class AlertRequestTest : public CommandRequestTest<CommandsTestMocks::kIsNice> {
+ public:
+  AlertRequestTest()
+      : mock_message_helper_(*MockMessageHelper::message_helper_mock())
```

Comment:
@okozlovlux, if you use `MockMessageHelper`, please, don't forget to verify and clear expectations.